nemo/core/classes/modelPT.py

```diff
@@ -211,6 +211,16 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self._memory_profile_started = False
         self._memory_profile_complete = False
 
+        from workload_inspector.bkg_runner import BackgroundRunner
+        from workload_inspector.torch.nsys_downstream import NsysDownstream
```

Comment:
what guarantees this package is available here?

Reply:
certain aspects of this package i.e., GPU wise perf analysis will be released publicly on github, and instructions can be added download them.